app/controllers/lessons_controller.rb

```diff
@@ -61,7 +73,7 @@ def destroy
 
   private
     def set_lesson
-      @lesson = lesson.find(params[:id])
+      @lesson = Lesson.find(params[:id])
```

Comment:
At some point we should add some sort of authorization library like `pundit` to make sure that `@lesson` belongs to `current_user`.